sources/api/apiserver/src/server/mod.rs

```diff
@@ -892,6 +978,12 @@ impl ResponseError for error::Error {
             UpdateLockOpen { .. } => StatusCode::INTERNAL_SERVER_ERROR,
             ReportExec { .. } => StatusCode::INTERNAL_SERVER_ERROR,
             ReportResult { .. } => StatusCode::INTERNAL_SERVER_ERROR,
+            DeserializeSettingsGenerator { .. } => StatusCode::BAD_REQUEST,
+            DeSerialize { .. } => StatusCode::BAD_REQUEST,
+            InvalidStrength { .. } => StatusCode::BAD_REQUEST,
+            DisallowStrongToWeakStrength { .. } => StatusCode::BAD_REQUEST,
+            Transpose { .. } => StatusCode::BAD_REQUEST,
+            Parse { .. } => StatusCode::BAD_REQUEST,
```

Comment:
I think these are technically internal server error, since they occur if we fail to parse strength from the datastore itself?